boringssl/src/test/resources/org/hyperledger/besu/nativelib/boringssl/secp256r1ecrecover.csv

```diff
@@ -0,0 +1,49 @@
+input,result,status,error, notes
+9b2db89cb0e8fa3cc7608b4d6cc1dec0114e0b9ff4080bea12b134f489ab2bbc0000000000000000000000000000000000000000000000000000000000000000976d3a4e9d23326dc0baa9fa560b7c4e53f42864f508483a6473b6a11079b2db1b766e9ceb71ba6c01dcd46e0af462cd4cfa652ae5017d4555b8eeefe36e1932,04e266ddfdc12668db30d4ca3e8f7749432c416044f2d2b8c10bf3d4012aeffa8abfa86404a2e9ffe67d47c587ef7a97a7f456b863b4d02cfc6928973ab5b1cb39,0,,pre-existing case from openssl
```

Comment:
these input lines are misleading.  we should expect recid to be 27 or 28, and we should subtract 27 (V_BASE) to get 0 or 1 for recid.  these inputs are not wrong for the tests, but because we are encoding recovery id directly as 00..00 or 00..01, these are not realistic precompile inputs.  

we should change the test parser to expect (recovery_id + V_BASE), and do the subtraction before calling the method (as we do in besu).  that way these inputs should be portable into besu r1 tests.